omniduct/databases/base.py

```diff
@@ -249,7 +227,7 @@ def add_template(self, name, body):
         self._templates[name] = body
         return self
 
-    def render_template(self, name_or_statement, context=None, by_name=False):
+    def render_template(self, name_or_statement, context=None, by_name=False, template=True):
```

Comment:
We should consider renaming this option at some point

Reply:
See above comment :).